python/sglang/srt/managers/controller/tp_worker.py

```diff
@@ -711,8 +768,20 @@ def handle_finished_requests(self, batch: Batch):
             else:
                 batch.reqs = []
 
+    def filter_out_inflight(self, batch: Batch):
+        if self.current_inflight_req is None:
+            return
+
+        unfinished_indices = list(range(len(batch.reqs)))
+        for i, req in enumerate(batch.reqs):
```

Comment:
Could you try something like `batch.reqs.index(self.current_inflight_req)` to find the index to remove?

Reply:
sorry, I was a little confused about the judging method between `is` and `==`